src/components/Background.tsx

```diff
@@ -0,0 +1,5 @@
+const Background = () => {
+  return <div className="chaotic-background"></div>;
```

Comment:
If you're not adding any children or content inside the `<div>` you can make the tag self-closing for a cleaner look:

```suggestion
  return <div className="chaotic-background" />;
```

Since your component is only returning a single JSX element, you can omit the return statement and the curly braces by using an implicit return.
```TypeScript
export const Background = () => <div className="chaotic-background" />;
```